### DISCUSSION TITLE
Suicide Squids

### DISCUSSION THREAD

**Discussion by ghost** [PurpurMC/Purpur#674] [__Dismissed__] [answered]
Add a option to have squids swim into players at a faster speed when noticed, and then explode. Same thing for the glow squid.

**Comment by encode42** [comment, marked as answer]
Though this would be an amazing feature for Purpur, it really would be easier to implement as a plugin.

The process would be simple, you'd do some logic for when a squid is "noticed" (range + rng?), pathfind to the player that's been noticed, then BOOM
